doc/contributing/documentation_guide.md

```diff
@@ -503,3 +513,66 @@ mention `Hash#fetch` as a related method, and `Hash#merge` might mention
 For methods that accept multiple argument types, in some cases it can
 be useful to document the different argument types separately.  It's
 best to use a separate paragraph for each case you are discussing.
+
+## Further Considerations
+
+### Case Treatments
+
+When a series of paragraphs documents _cases_ that form a _partition_,
+consider using bullets (unordered items) to emphasize that structure;
+such structures may be nested.
+
+The documentation for Array.new is organized thus (some text omitted)
+
+```
+Returns a new +Array+ object:
+
+- With no block given:
+
+    - With no argument given, returns ...:
+    - With argument +array+ given, returns ...:
+    - With numeric argument +size+ given, returns ...:
+
+- With a block given, returns ...:
+
+Raises TypeError if ....
+```
+
+### Trivial \Method Behaviors
+
+For a trivial method behavior,
+it's usually best not to include that behavior in the discussion;
```

Comment:
If it isn't worth discussing, I'm not sure we should encourage providing a code example.  For Array.new, I would drop at least two of the examples with explicit size 0.